src/NUnitFramework/tests/TestXmlFile1.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Right now, the top level root node is hard coded to *data*. The child nodes can have various names, so long as the column count matches that of the method's argument count